src/Http.ts

```diff
@@ -136,17 +141,28 @@ export default class Http implements HttpController {
         return modified_result ? parser.parseResult(modified_result) : parser.parseResult(result)
     }
 
-    public async update(config: EntityUpdateConfig, entity: string) {
+    public async update(config: EntityUpdateConfig | EntityUpdateConfig[], entity: string) {
         await this.client.account_promise
         const url = this.getRequestUrl('mutate', entity)
         const options = await this.getRequestOptions('POST', url)
 
-        const update_operation = {
-            update: config.update,
-            update_mask: getUpdateMask(config.update),
+        if (Array.isArray(config)) {
+            const operations = config.map(operation => {
+                const update_operation = {
+                    update: operation.update,
+                    update_mask: getUpdateMask(operation.update),
+                }
+                update_operation.update.resource_name = this.buildResourceName(entity, operation.id)
+            })
```

Comment:
Missing return in map, elements in the `operations` array are going to be undefined

Reply:
This should be fixed now.